src/Model/CustomerInterface.php

```diff
@@ -205,9 +205,8 @@ public function getIdEncoded(): ?string;
     public function setIdEncoded(?string $idEncoded);
 
     /**
-     * Return type bool is deprecated and will be removed in version 4
      *
-     * @return Consent|bool|null
+     * @return Consent|null
```

Comment:
since this is already a BC break, we can enable the return type hint.

Reply:
Thanks for the reminder. I have added the return type and removed the doc block!